src/Main.java

```diff
@@ -1,5 +1,70 @@
+public class Author {
+    private String name;
+    private String lastName;
+
+    public Author(String name, String lastName) {
+        this.name = name;
+        this.lastName = lastName;
+    }
+    public String getName() {
+        return this.name;
+    }
+    public String getLastName() {
+        return this.lastName;
+    }
+}
+public class Book {
+    private String nameBooks;
+    private int agePublication;
+    private String author;
```

Comment:
Автор это автор, а не строка) Соотвественно тип должен быть Author,  а не String
```suggestion
    private Author author;
```